client/src/pages/Questions/index.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+
+import Tabs from './components/Tabs'
+
+export default function QuestionsPage(props) {
```

Comment:
name the function Page4 
so the name can follow one pattern with the other pages

Reply:
So you need to change the issues title, from ouestionsPage to Page4!

Reply:
No i mean the function name

Reply:
ah, okay